src/test/kotlin/com/ampnet/reportservice/service/JpaServiceTestBase.kt

```diff
@@ -30,13 +34,17 @@ import java.util.UUID
 abstract class JpaServiceTestBase : TestBase() {
 
     protected val userUuid: UUID = UUID.fromString("89fb3b1c-9c0a-11e9-a2a3-2a2ae2dbcce4")
+    protected val secondUserUuid: UUID = UUID.randomUUID()
+    protected val thirdUserUuid: UUID = UUID.randomUUID()
     protected val walletUuid: UUID = UUID.fromString("d3499ace-ee85-11ea-adc1-0242ac120002")
     protected val projectUuid: UUID = UUID.fromString("979dd8c5-765d-49a4-b64d-142a3c55f4df")
     protected val userWalletHash: String = "user wallet hash"
     protected val projectWalletHash: String = "project wallet hash"
     protected val mintHash: String = "mint"
     protected val burnHash: String = "burn"
     protected val txHash: String = "tx_hash"
+    protected val logo = "https://ampnet.io/assets/images/logo-amp.png"
```

Comment:
I wouldn't rely on this image. Who knows whether it will be there in the future, but how you like it.